learn/templates/404.html

```diff
@@ -1,4 +1,619 @@
+
+                    <li class="e1">e</li>
+    
+                    <li class="e2">a</li>
+    
+                    <li  class="e3">s</li>
+    
+                    <li  class="e4">y</li>
+    
+                    <li>v</li>
+    
+                    <li>f</li>
+    
+                    <li>u</li>
+                <li  class="e5">l</li>
+
+                <li class="e6">e</li>
+
+                <li class="e7">a</li>
+
+                <li class="e8">r</li>
+
+                <li class="e9">n</li>
+
+                <li class="e10">i</li>
+
+                <li class="e11">n</li>
+
+                <li class="e12">g</li>
+
+                <li>t</li>
+
+                <li>a</li>
+
+                <li>x</li>
+
+                <li>l</li>
+
+             
+
+                <li>y</li>
+
+                <li>y</li>
+
+                <li>w</li>
+
+                <li>v</li>
+                
+                <li>b</li>
+
+                <li>o</li>
+                <li class="one">4</li>
+
+                <li class="two">0</li>
+
+                <li class="three">4</li>
+                <li>q</li>
+
+                <li>d</li>
+
+                <li>y</li>
+
+                <li>p</li>
+
+                <li>a</li>
+
+
+                <li>v</li>
+
+                <li class="four">p</li>
+
+                <li class="five">a</li>
+
+                <li class="six">g</li>
+
+                <li class="seven">e</li>
+                <li>j</li>
+
+                <li>a</li>
+
+                <li class="eight">n</li>
+
+                <li class="nine">o</li>
+
+                <li class="ten">t</li>
+
+                <li>s</li>
+
+                <li>c</li>
+
+                <li>e</li>
+
+                <li>w</li>
+
+                <li>v</li>
+
+                <li>x</li>
+
+                <li>e</li>
+
+                <li>p</li>
+
+                <li>c</li>
+
+                <li>f</li>
+
+                <li>h</li>
+
+                <li>q</li>
+
+                <li>e</li>
+
+                <li class="eleven">f</li>
+
+                <li class="twelve">o</li>
+
+                <li class="thirteen">u</li>
+
+                <li class="fourteen">n</li>
+
+                <li class="fifteen">d</li>
+
+                <li>s</li>
+
+                <li>w</li>
+
+             
+            </ul>
+        </div>
+
+        <div id="main-content">
+            <h1>We couldn't find what you were looking for.</h1>
+
+            <p>Unfortunately the page you were looking for could not be found. It may be
+                temporarily unavailable, moved or no longer exist.</p>
+
+            <p>Check the URL you entered for any mistakes and try again. Alternatively, search
+                for whatever is missing or take a look around the rest of our site.</p>
+
+            <div id="navigation">
+                <a class="navigation" href="{% url 'index' %}">Home</a><a class="navigation" href="">Go Back</a>
```

Comment:
Go back URL is not working as expected.
It should go to previous valid URL, but it is reloading the same page again.

Reply:
I have'nt added any href to go back url

Reply:
Added Go back URL.